app/jobs/index_annotation_job.rb

```diff
@@ -11,7 +11,7 @@ def perform(annotation)
   private
 
   def index(annotation)
-    manifest_uri = annotation.data['on'].first['within']['@id']
+    manifest_uri = JSON.parse(annotation.data)['on'].first['within']['@id']
```

Comment:
I find something suspect about all this. Annotot says it is serializing and deserializing the `data` attribute for us:

https://github.com/mejackreed/annotot/blob/master/app/models/annotot/annotation.rb#L6

Is it possible we're just storing strings instead of actual structured data?

Reply:
Perhaps it is? https://github.com/mejackreed/annotot/blob/master/app/controllers/annotot/annotations_controller.rb#L19